README.md

```diff
@@ -0,0 +1,350 @@
+
+[](https://github.com/TamaraDimov/vespa-rentals-backend)
+
+<img src="">
+
+## 📄 Project Documentation <a name="project-documentation"></a>
+
+_Here you can find a detailed documentation for the API used in the project, including endpoints, request/response formats, and authentication methods. Developers can use this documentation to integrate our service into their applications and build custom integrations._
+
+- Front-end GitHub repository [link](https://github.com/TamaraDimov/vespa-rentals-frontend)👈
+- Back-end API [link](https://github.com/TamaraDimov/vespa-rentals-backend) 👈
+- API Documentation [link](https://github.com/TamaraDimov/vespa-rentals-backend) 👈
+
+## ✒ Kanban Board <a name="kanban-board"></a>
+
+_We used a Kanban board to track the progress of tasks and issues. The board is designed to help visualize the status of work items, identify bottlenecks, and ensure that work is flowing smoothly._
+
+### Kanban Board [link](https://github.com/users/TamaraDimov/projects/4/views/1?layout=board)
+
+  <details>
+    <summary>Initial State</summary>
+
+<img src="./init kanban .png" alt="logo" width="auto"  height="auto" />
+
+  </details>Final State;</summary>
+
+  <details>
+    
+<img src="" alt="logo" width="auto"  height="auto" />
+
+  </details>
+</details>
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+_To get a local copy up and running follow these simple steps._
+
+### Prerequisites
+
+_In order to run this project you need:_
+
+- Mac or PC
+- Install node.js
+- Use npm as package manager.
+- Understanding of React and Redux
+
+## Front end
+
+### Setup
+
+- _Clone this repository to your desired folder: https://github.com/TamaraDimov/vespa-rentals-frontend.git_
+
+```sh
+ git clone
+```
+
+- _Navigate into the cloned folder_
+
+```sh
+ cd vespa-rentals-frontend
+
+```
+
+### Install
+
+Install thr project dependencies with:
+
+```
+  npm install
+```
+
+### Usage <a name="usage"></a>
+
+To run the project, execute the following command:
+
+```
+  npm start or npm s
+```
+
+### Run tests
+
+To run the tests, run the following command:
+
+```
+  npm test
+```
+
+### Deployment
+
+You can deploy this project using:
+
+```
+   npm run build
+```
+
+## Back end
+
+### Setup
+
+- _Clone this repository to your desired folder: https://github.com/TamaraDimov/vespa-rentals-backend.git_
+
+```sh
+ git clone
+```
+
+- _Navigate into the cloned folder_
+
+```sh
+ cd vespa-rentals-backend
+
+```
+
+### Install
+
+Install thr project dependencies with:
+
+```
+  bundle install
+  rails db:create
+  rails db:migrate
+```
+
+### Usage<a name="usage"></a>
+
+To run the project, execute the following command:
+
+```
+  rails server or rails s
+```
+
+### Run tests
+
+To run the tests, run the following command:
+
+```
+  rspec ./spec/models
+  rspec ./spec/controllers
```

Comment:
- [ ] When I run the tests according to instructions, this happens:

![image](https://github.com/TamaraDimov/vespa-rentals-backend/assets/57421823/d05cf5c7-fd90-4b56-9bd8-94aeef66984c)

![image](https://github.com/TamaraDimov/vespa-rentals-backend/assets/57421823/83b7bd04-de9f-4200-b2e3-024742e20d1c)

This is because the `./spec/models folders` does not exist, and the test file inside `./spec/controllers` has an error.

Kindly fix your tests and/or your instructions so they all pass. 👍